Cargo.toml

```diff
@@ -25,6 +25,7 @@ pub-iterator-type = "0.1"
 self_cell = "0.10.0" 
 serde = { version = "1.0", features = ["derive"] }
 smartstring = { version = "0.2", features = ["serde"] }
+chrono = { version = "*", features = ["serde"] }
```

Comment:
You can't depend on "*", you must give a version (else you can't publish to crates.io).

I prefer to not depend on chrono, any time library has a constructor for a timespan, so better to gives the `i64` that any such lib expect, it will be more flexible, and also smaller.

Reply:
Makes sense, I'll rewrite to just use the `i64`!